widgets/DynamoDB-batchsource.json

```diff
@@ -1,6 +1,13 @@
4UgOROn7XVT4/5Clsx+/H7I131v1tt5N870cI6AzvGuElyowbRGix7SikJvFOrspVYawWokm5xZKATgFfLZdEND4czER145NUmCAw+moenz2w46Qi2563JN5tvzzizehPGFotRid22mxwoAveIFktMTJA2FotZpaDl9m8xcPuMP9n8m5Qp0BiyEzs3D5cs12Ymu2iy5vX6yrm8DtcpwrdQklPsGs1OqWpT51YfIMklc+3vAEx1U/D1bNChZounQOdGtiB2DbP17oyMC0UVv0KChRq9PJlLTn3ycvvnvbQB4VV6529cFrPmhoYfm9ID+eTC8ggGwh8C+zNCY2yZIHb0PSh09c7npa1cz8ZEHYAi3vPP2O7APlHxqbO7q3+KE7F37lPXsWl0/fPBRSP68dk2XFDF8t6E1vU12rZZq+vL76Zo2AeAns9NX71/U9j82fvrYXxY+YvKuVSbDa37MF9t0f3TzF69J4b4RM8bpvSzXbcKeF4YCfduIL7Z5pJiermuHUa1r5DbCyx+DfbXaZABz7wa799X1NtjzF4/u+VfT7h2AH4fkEtiTjcz+MJmmx8H+sfXQu7+qOmlUw8meJWydu2O8Ptg+PaAztZgYdAadiUFnYtCZGHQmBp2JQWdqI+gaYbsbbC4jJrAbW/5Jd8JokGCqlpYM4TEwjVRel9L8bZVwvPVOiodruFmME0EQ6fUILjfdjsQLAhFEN/073iPAhRH076KrNaFrWlFQMpPzwCqgtGUg6T63Muh6tz5l0hO2qTX1bubBQsDTAuCmhQMLSaWwcPBepAUEjotQWAQXKWoeRSHu/3KCS+J4MQh/w5svLqOhnwELl1/j4ruNa35aVQQ1M8V86KrZUwTDPeyFWr8SLdu8cOMHL0qEd3lmeZc3JXqC1wW3rwCvRThe1QXqvnKGyRKxlTNMbMjGxKAzMehsnH6noPN2tjRwpgqzLLNY0LHmBRjiwZCPwx48mCAFTm765s9T0HvnoEePz3jZ1uyQ7XP1+Q5R4T1hobQme41x+fwxpXxMawNeAAzG4aXJHIFO1ghCaXy+0rgdLatKs0VVyAFcEYZqHvgeblZcvlZx0NLJGY4TFMHX1dhq2CWFY4VZufJsneTrsM2MXDTiJ6KrPKECMJdPrvACzsoBXHjPNbCnb3RsiiOy0mOqeze3KvC0DHMrhIJZXLSlqH2WQPcavASadeSYGHQmBp2JQWdi0JkYdAadiUFnYtCZ2hA67k+XWVY5QngDo/n96eOnj30Hkk+D/Yblqa31Dtj2i0f3VI23U9MENoC/AslT64cP/gHSX1cpbGzdu4lS8qlEMZc8NPLqN16t9Tv13rW4WaOLYTJYqpxLp29d8hcSt/Bp0Ffq+a7Iss9hjbYiZzPxESk3M+ZvdA1CLStndkGC93P7wXaybLcItqYWspNX+Nz0VS8+9H+ZvgWctkJ6Q3D74pC+32xN/yvLcgvb7GJBzN2+kc/E/ydhfJxV9BSaIAUqz4YNm+rewf14inPXlzy0nz0Fuo62WlFIanaKpGZKz4RPTMc9SjFfdSWSJzqo+nu2qITjqj5zzoA2XeMwFhuNx1Y+ggv4SvFX2vt57yupFAJkIfIDDQGiLhkArQkctzgF+h9IuwNdflLjZJspHTkaAhuM5GZKF9omkR1WaJhpAL5E+VnuWJtz6caD/WAsm9DgDntGdrhzJC8TLS8Tki9HncIYLuUAPU6O4XKnq8YYLlOlqA0AGh/eX8fGxTWqN0d8XffkvV0bMcS5v+7CYo8aoNLmQJEX4pnQECAVT4Apb//RJUZRTFC4xkRrQvEuD8HgfKIn5PBoTSs2CTlq84VgUZNQzNijEKgCTyZGp+bb43wmZejvuUM9KrTfeY7jm4rG6JjJmcVNQjJ1yzbXNWPCb+A6en/vfQkp3Iewvc2ej83I2VyCFMSx9yy4dd22lOm/w0Vw5Xlft6RVwmez+y/1g6ZRlftJpGeAJIv+ZFHRdw+hESG6VB4DAIMJuBmpkKHx07EQaCx++urttcdHQ2cj6I51vfPbtlJjUypRFGJz6EsKAOHcfiK4cVTRAx2zLK39Wh4KgJJve9DeYJhEKehBEgibtz3K1DYdh18CjsF9XTTOugYFgHoBOdMelGF8HYx0k2gvgI4NEo8/aMll1AL9BbCX9W/73YTzRgkPho9bKjUBUAjkVEttW8btxh3dfVCjBwD0ABHdhs8+vlA172sM3PMLeLm/RphZMbKx8WEFXA+Cn+8IqorjQCPYSKyfuu6Orj6677xR1RG4B3Xy4tE9B6oxrdW9fxsMg4E+YoYL5N3g9sAwq9D1V7zAGrcWLRe6anTZ2D4Ho91LHsVmks6WORFdajoKajs+wO482L2G1vS1nAB0/rATyAscdhGtb6J5jnT3rqc9bl8wYshv1FjTMSjfLqjlt0vOUtMHehn8PZBcIMueYGgW9IpYXLYlQtA7AfjlhRZSx4cHjp8+hifG6K1sHbw9hBz2LgZeU+ey3l8B8OcgeZrlty30NAA/V/eIopFfAvCvrTGMY/Ou5ugIAH+toWFkEz/6A7DXV+pvMR6GC/P9hw3PHTT6RajtCPdAuUfPZJ4wvw9ALddMh14Gnyt37EYZC1M0Wu645Zo5SdML0QA8Lq7HiL4JxsRQYf4+DsDjzZ5Il9WHAB5DbT8Bxu6dGiPM1ycA+CU9TqbbklMA/x4phd1m0l/PAvD39DqZruuMAfwJzh04xRjpp4BPegeAn9DznLovLr/x+xe/BgluejzNkDUlzL9dZ1985Em9T1zX3Hu9Gtr3Cu5yPQw2rGvpb+25d4R9+M3ntl9o9AS63nCxC/wWhd40bFtB1xt+i0HXDbYtoesFv0Wg6w7b1tCbhe9w6IbBdgT0RuE7FLrhsB0FvV74DoNuGmxHQq8VvkOgmw7b0dCrwbc5dMtgtwT01eDbFLrlsFsK+iL4D0NyBKDvthF028DWDTpT6+n/AgwAJLG992JJZ3kAAAAASUVORK5CYII="
```

Comment:
can we add the icon to icons folder and have the cdap-maven plugin package it?

Reply:
+1, though I think the maven plugin still needs to be published, only snapshot versions are available.